delft3dworker/admin.py

```diff
@@ -73,3 +82,99 @@ class SearchFormAdmin(GuardedModelAdmin):
 class TemplateAdmin(GuardedModelAdmin):
     pass
 
+
+@admin.register(GroupUsageSummary)
+class GroupUsageSummaryAdmin(admin.ModelAdmin):
+    """
+    # Following example available at:
+    # https://medium.com/@hakibenita/how-to-turn-django-admin-into-a-lightweight-dashboard-a0e0bbf609ad
+    """
+    change_list_template = 'delft3dworker/group_summary_change_list.html'
+    # Filter by time period
+    list_filter = (('user__scene__workflow__stoptime', DateRangeFilter),)
+
+    def changelist_view(self, request, extra_context=None):
+        """
+        Displays summary of usage organized by group
+        """
+        response = super(GroupUsageSummaryAdmin, self).changelist_view(
+            request,
+            extra_context=extra_context,
+        )
+        try:
+            qs = response.context_data['cl'].queryset
+            qs = qs.exclude(name='access:world').order_by('name')
```

Comment:
We could clarify (in a comment) why we filter out this group. This does seem to imply that scenes in *world* are not billed?